Uchu.World/Handlers/Commands/OperatorCommandHandler.cs

```diff
@@ -236,5 +237,57 @@ public string Target(string[] arguments, Player player)
 
             return $"Target: {baseCombatAiComponent.Target}";
         }
+
+
+        [CommandHandler(Signature = "execute", Help = "Execute a command as another player", GameMasterLevel = GameMasterLevel.Admin)]
+        public async Task<string> Execute(string[] arguments, Player executor)
+        {
+            List<Player> players = new List<Player>();
+            if (arguments[0] == "*") {
```

Comment:
The code in this block doesn't adhere to the overall Uchu / C# style, we generally do:

```
if ()
{
    ... stuff
}
else
{
    ... other stuff
}
```

and

```
foreach ()
{
    ... stuff
}
```